test/fixture/async-await.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 const test = require('../../');
+const Promise = require('bluebird');
```

Comment:
Yes, it is lame to include polyfills in test files - but it is bad to include global polyfills either (because they leak to production code).